tests/quick_mode_removal_test.py

```
@@ -0,0 +1,312 @@
+"""
+Tests for removal of quick mode functionality and checkpoint frequency updates.
+"""
+
+import pytest
+import asyncio
```

Ruff (0.12.2)

`asyncio` imported but unused

Remove unused import: `asyncio`

(F401)